Doc/library/sys.rst

```diff
@@ -1022,19 +1022,22 @@ always available.
 
    .. attribute:: hash_info.modulus
 
-      The prime modulus P used for numeric hash scheme
+      The prime modulus P used for numeric hash scheme.  Available also as
+      ``_PyHASH_MODULUS`` macro.
```

Comment:
Usually we don't add references to the C API in the Python documentation. Moreover, usually we don't document the private API.

Reply:
Redo in Doc/c-api/hashing.rst per [suggestion](https://github.com/python/cpython/issues/111389#issuecomment-1784940869) in the issue thread.